internal/liqonet/route-operator.go

```diff
@@ -196,7 +219,7 @@ func (r *RouteController) createAndInsertIPTablesChains() error {
 	} else {
 		log.Info("created", "chain", LiqonetPreroutingChain, "in table", NatTable)
 	}
-	r.IPTablesChains[LiqonetPostroutingChain] = liqonetOperator.IPTableChain{
+	r.IPTablesChains[LiqonetPreroutingChain] = liqonetOperator.IPTableChain{
```

Comment:
Out of curiosity: Why do we moved to PreRoutingChain?

Reply:
it was a little bug where we saved the _LiqonetPreroutingChain_ as the _LiqonetPostroutingChain_ in the _IPTablesChains_ table. Looking at line 217 it is creating the _LiqonetPreroutingChain_.